internal/helpers/prometheus_helper.go

```diff
@@ -368,18 +369,18 @@ func (mrs *MonitoringRuleSet) createMagicMultiBurnRateAlert(burnRates []monitori
 	if severity == "page" {
```

Comment:
We construct both severity page and ticket in this one function instead of this `if else if magic`